tools/online_editor/src/index.ts

```diff
@@ -23,12 +23,17 @@ import {
   ErrorAction,
   MonacoServices,
   MessageTransports,
+  Message,
+  RequestMessage,
+  ResponseMessage,
 } from "monaco-languageclient";
 import {
   BrowserMessageReader,
   BrowserMessageWriter,
 } from "vscode-languageserver-protocol/browser";
 
+import { ProxyReader } from "./proxy.ts";
```

Comment:
The CI complains about the .ts extension.  Maybe this would be better?  (untested, i have no clue what i'm doing)

```suggestion
import { ProxyReader } from "./proxy";
```